<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoofdScherm</class>
 <widget class="QMainWindow" name="HoofdScherm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoofdScherm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushShaderSwitch">
      <property name="text">
       <string>A Shader</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="MijnGLWidget" name="glScherm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelFPS">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MijnGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>mijnglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
